code/datums/components/dejavu.dm

```diff
@@ -138,12 +138,12 @@
 	no_rewinds_message = "\"Rewind complete. You have arrived at: 10 seconds ago.\""
 
 /datum/component/dejavu/timeline/rewind()
-	playsound(get_turf(parent), 'sound/items/modsuit/rewinder.ogg')
+	playsound(parent, 'sound/items/modsuit/rewinder.ogg').play()
 	. = ..()
 
 /datum/component/dejavu/wizard
 	rewind_message = "Your temporal ward activated, pulling you through spacetime!"
 
 /datum/component/dejavu/wizard/rewind()
-	playsound(get_turf(parent), 'sound/items/modsuit/rewinder.ogg')
+	playsound(parent, 'sound/items/modsuit/rewinder.ogg').play()
```

Comment:
any particular reason these no longer call `get_turf(parent)`?

Reply:
because you should not be using a turf as a sound source unless the source of that sound is actually the turf